File: Objects/objects.js
var Person= function(age, name){
	this.name=name;
	this.age=age;
};
Person.prototype.getName = function() {
	return this.name;
};
Person.prototype.getAge=function(){
	return this.age;
}




var p=new Person("Sub", 12);
console.log(p.getName());
console.log(p.getAge())

File: Objects/objects.py
class Person:
	def __init__(self, age, name):
		self.age=age
		self.name=name
	def getAge(self):
		return self.age
	def getName(self):
		return self.name

class Animal:
	def __init__(self):
		

a = Person(10, "hi")

print(a.getAge())
print(a.getName())